recipes/llvm/all/conanfile.py

```diff
@@ -0,0 +1,125 @@
+from conans import ConanFile, tools, CMake
+from conans.tools import Version
+from conans.errors import ConanInvalidConfiguration
+import os, shutil, glob
+
+projects = [
+    'clang',
+    'clang-tools-extra',
+    'compiler-rt',
+    'debuginfo-tests',
+    'libc',
+    'libclc',
+    'libcxx',
+    'libcxxabi',
+    'libunwind',
+    'lld',
+    'lldb',
+    'mlir',
+    'openmp',
+    'parallel-libs',
+    'polly',
+    'pstl'
+]
+
+default_projects = [
+    'clang',
+    'compiler-rt'
+]
+
+class Llvm(ConanFile):
+    name = 'llvm'
+    description = 'The LLVM Project is a collection of modular and reusable compiler and toolchain technologies'
+    url = 'https://github.com/conan-io/conan-center-index'
+    homepage = 'https://github.com/llvm/llvm-project'
+    license = 'Apache 2.0'
```

Comment:
```suggestion
    license = 'Apache-2.0'
```

https://spdx.org/licenses/Apache-2.0